maven_iuvs/download.py

```diff
@@ -625,7 +626,7 @@ def sync_integrated_reports(sdc_username, sdc_password, check_old=False):
     """
 
     print("syncing Integrated Reports...")
-    
+
```

Comment:
thanks for the PEP8 check!

Reply:
Oops it accidentally caused during debugging, but I guess it does not violate anything. I often insert import pdb; pdb.set_trace() (similar to the stop command in IDL) to check values in variables, so I could delete the spaces  when I delete the pdb command.

Reply:
This whitespace was in my original commit, thanks for fixing it!